client/my-sites/plans/jetpack-plans/upsell/index.tsx

```diff
@@ -1,20 +1,25 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
I guess it's not the scope of this PR - however, I see we disable this eslint rule in lots of files - should we consider disabling it for this project altogether instead?

Reply:
The rule is too restrictive in my opinion. I ignored it because it would required all classes to start with `upsell__`, which could easily conflict with other components in this codebase.

The proper fix would be to rename the folder `upsell` by something like `jetpack-plans-upsell`, but that's quite verbose given the name of the parent folder. Maybe that rule isn't meant to be used with deep folder structures, but that's how the codebase evolved.

That's something that could be brought up indeed.